src/components/Template/Template.mdx

```diff
@@ -47,72 +76,271 @@ import {
   TemplateBottom
 } from "@nypl/design-system-react-components";
 //...
+<Template sidebar="...">
+  <TemplateBreakout>...</TemplateBreakout>
+  <TemplateTop>...</TemplateTop>
+  <TemplateSidebar>...</TemplateSidebar>
+  <TemplateMain>...</TemplateMain>
+  <TemplateBottom>...</TemplateBottom>
+</Template>
+`}
+  language="jsx"
+/>
 
-<>
-  <SkipNavigation />
-  <Template sidebar="...">
-    <TemplateBreakout>...</TemplateBreakout>
-    <TemplateTop>...</TemplateTop>
-    <TemplateSidebar>...</TemplateSidebar>
-    <TemplateMain>...</TemplateMain>
-    <TemplateBottom>...</TemplateBottom>
-  </Template>
-</>
-`} language="jsx" />
+## Component Props
```

Comment:
There are no descriptions to the props

Reply:
Yes, I also noticed that, but I was not sure why. I will see what I can do.

Reply:
You might need to `export default` the main `Template` component.

Reply:
I made a change, but the descriptions and expected values are still not showing. This was the update:
```
export {
  TemplateBreakout,
  TemplateTop,
  TemplateMain,
  TemplateMainNarrow,
  TemplateSidebar,
  TemplateBottom,
};

export default Template;
```
Previously, `Template` was in the blanket `export` statement with all the others.

Reply:
I think `Template` needs to be in both, the named export and the default export.

Reply:
That didn't work either. Anyone else want to take a stab at it?

Reply:
:(